src/lib.rs

```diff
@@ -308,7 +316,22 @@ impl<C> PipelineBuilder<C> {
     /// ```
     pub fn with_service_name<T: Into<String>>(mut self, name: T) -> Self {
         self.service_name = name.into();
```

Comment:
```suggestion
```

Since we're now setting it using an `sdk::Config`, we don't need to set it as a field on `PipelineBuilder`.

Reply:
whoops